core/src/org/samtools/htsjdk/core/Locatable.java

```diff
@@ -0,0 +1,140 @@
+package org.samtools.htsjdk.core;
+
+import java.util.OptionalLong;
+
+/**
+ * Represents the location of a sequence with respect to another sequence (reference).
+ *
+ * <p>
+ * This interface represents a location or "mapping" with respect to a reference sequence. Examples
+ * of this abstraction are features located on the genome or alignment between two sequences (query
+ * with respect to a reference).
+ * </p>
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)

```

Comment:
Do we want author's tags in the new code?  My feeling has been that they rapidly become out of date and are handled better by git history.   Do others feel differently?

Reply:
Was the default in my IDE - will remove!

Reply:
Done.